packages/vaex-core/vaex/functions.py

```diff
@@ -437,6 +437,35 @@ def dt_month_name(x):
     import pandas as pd
     return pd.Series(_pandas_dt_fix(x)).dt.month_name().values.astype(str)
 
+@register_function(scope='dt', as_property=True)
+def dt_quarter(x):
+    """Extracts the quarter from a datetime sample.
+
+    :returns: an expression containing the number of the quarter extracted from a datetime column.
+
+    Example:
+
+    >>> import vaex
+    >>> import numpy as np
+    >>> date = np.array(['2009-10-12T03:31:00', '2016-02-11T10:17:34', '2015-11-12T11:34:22'], dtype=np.datetime64)
+    >>> df = vaex.from_arrays(date=date)
+    >>> df
+      #  date
+      0  2009-10-12 03:31:00
+      1  2016-02-11 10:17:34
+      2  2015-11-12 11:34:22
+
+    >>> df.date.dt.month_name
```

Comment:
```suggestion
    >>> df.date.dt.quarter
```

Reply:
If you accept this change we can merge this, sounds good?

Reply:
Yes, looks good!